p4-evaluate-news-nlp/src/client/js/formHandler.js

```diff
@@ -1,16 +1,42 @@
-function handleSubmit(event) {
-  event.preventDefault();
+import { checkForName } from "../js/nameChecker";
+let formText;
```

Comment:
why initialize here?

Reply:
done in [fa51653](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/8/commits/fa516536cb6f3f2079108daa9df3466a2e84a077)